.taplo.toml

```diff
@@ -0,0 +1,19 @@
+include = ["**/Cargo.toml"]
+
+[formatting]
+align_entries = false
+align_comments = false
+array_trailing_comma = true
+array_auto_expand = true
```

Comment:
Maybe false for this, otherwise we'd have something like:
```
scale-info = { version = "2.1.1", default-features = false, features = [
	"derive",
] }
```
which is a bit werid

Reply:
This is a bit hard. **array_auto_expand** controls all arrays. If it is set to *false*, e.g:
```
std = [
	"codec/std",
	"sp-std/std",
	"sp-runtime/std",
	"sp-io/std",
	"sp-core/std",
	"sp-arithmetic/std",
	"teerex-primitives/std",
	"pallet-teerex/std",
	"frame-support/std",
	"frame-system/std",
	"frame-benchmarking/std",
]
```
will become
```
std = ["codec/std", "sp-std/std", "sp-runtime/std", "sp-io/std", "sp-core/std", "sp-arithmetic/std", "teerex-primitives/std", "pallet-teerex/std", "frame-support/std", "frame-system/std", "frame-benchmarking/std"]
```
😢

Reply:
What about setting `array_auto_collapse` also to false?